install/deploy_management_nodes.md

```diff
@@ -725,6 +725,37 @@ The LiveCD needs to authenticate with the cluster to facilitate the rest of the
 
 After the NCNs are booted, the BGP peers will need to be checked and updated if the neighbor IP addresses are incorrect on the switches. Follow the steps below and see [Update BGP Neighbors](../operations/network/metallb_bgp/Update_BGP_Neighbors.md) for more details on the BGP configuration.
 
+**`IMPORTANT:`** If the management switches are using CANU generated 1.0 (preconfig) this procedure should be skipped.
```

Comment:
Remove the backticks from IMPORTANT: to be consistent with the rest of docs-csm.